modules/popup-stack/stories/stories.tsx

```diff
@@ -0,0 +1,15 @@
+/// <reference path="../../../typings.d.ts" />
+import React from 'react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {PopupStack} from '@workday/canvas-kit-popup-stack';
+import README from '../README.md';
+
+export default {
+  title: 'Components|Popups/Stack Manager',
+  decorators: [withReadme(README)],
+};
+
+export const MixedPopupTypes = () => {
+  return <div />;
```

Comment:
What are the plans for this story?

Reply:
My plan is to test functionality of a few popup types - modals, selects, tooltips and a special Window component to show that it works correctly. Similar to this:

![2020-05-18 13-38-26 2020-05-18 13_40_27](https://user-images.githubusercontent.com/338257/83302709-0c8fe500-a1b9-11ea-9b69-4280695b2a2a.gif)